docs-source/content/security/openshift.md

```diff
@@ -5,14 +5,111 @@ weight: 7
 description: "OpenShift information for the operator"
 ---
 
-#### OpenShift `anyuid` security context
-
-The Docker images that Oracle publishes default to the container user
-as `oracle`, which is UID `1000` and GID `1000`. When running the
-Oracle images or layered images that retain the default user as
-`oracle` with OpenShift, the `anyuid` security context constraint
-is required to ensure proper access to the file system within the
-Docker image. This means that the administrator must:
+#### Security requirements to run WebLogic in OpenShift
+
+WebLogic Server Docker images obtained from Oracle Container Registry have an `oracle` user with
+UID 1000 and it is in a group which is also called `oracle` and has GID 1000.
+The WebLogic Kubernetes Operator Docker images use the same UID and GID.
+
+OpenShift, by default, allocates a high, random UID for each container and this
+UID may not have the necessary permissions to run the programs in the container.
+
+Since August 2020, the standard WebLogic Server images that are published in
+Oracle Container Registry set the file system group ownership to
+allow the high UIDs allocated by OpenShift to have the necessary permissions.
+
+Here is an excerpt from a standard WebLogic [Dockerfile](https://github.com/oracle/docker-images/blob/master/OracleWebLogic/dockerfiles/12.2.1.4/Dockerfile.generic#L89)
+that demonstrates how the file system group ownership is configured in the standard WebLogic images:
+
+```bash
+# Setup filesystem and oracle user
+# Adjust file permissions, go to /u01 as user 'oracle' to proceed with WLS installation
+# ------------------------------------------------------------
+RUN mkdir -p /u01 && \
+    chmod 775 /u01 && \
+    useradd -b /u01 -d /u01/oracle -m -s /bin/bash oracle && \
+    chown oracle:root /u01
+
+COPY --from=builder --chown=oracle:root /u01 /u01
+```
+
+However, if you build your own image, or obtain an image from another source, it
+may not have the necessary permissions.  You may need to configure similar file
+system permissions to allow your image to work in OpenShift.
+
+Alternatively, you may choose to configure OpenShift to allow use of UID 1000.  This
+can be done using a Security Context Constraint.  Oracle recommends that you define
```

Comment:
Security Context Constraint -> Security Context Constraint (SCC)
I checked the capitalization of this term in the OpenShift docs and this is what I found: it does not need to be capitalized. It can be capitalized prior to providing its abbreviation, SCC, but otherwise, can be used lower case. (They are not 100% consistent in their docs, but that's my takeaway from them.)